server/index.ts

```diff
@@ -0,0 +1,6 @@
+import { services } from './services'
+import createApp from './app'
+
+const app = createApp(services)
+
+export default app
```

Comment:
previously creating object graph and app creation happened in the same place now separated out